eng/pipelines/templates/jobs/ci.yml

```diff
@@ -72,6 +72,32 @@ jobs:
       name: azsdk-pool-mms-win-2022-general
       vmImage: MMS2022
     steps:
+      # Skip sparse checkout for the `azure-sdk-for-<lang>-pr` private mirrored repositories
```

Comment:
Should we move this whole step list into eng/common so we can call a simple template in all these PRs?

Reply:
Yeah, it'd be good to turn this into a shared script. The only thing we may need to add is the ability to configure the checkout `StepList`. It seems that Java, JS, and Python reserve the `Compliance` job to just CredScan but the .NET repo has additional checks within `Compliance` that may require a larger checkout scope.